assets/sass/patterns/molecules/social-links.scss

```diff
@@ -55,7 +55,6 @@
       @include margin(2 0 0 7);
     }
   }
-
 }
 
 @media only screen and (min-width: #{get-rem-from-px(730)}rem) {
```

Comment:
Can we not use magic numbers and use the variables ```$bkpt-site--x-medium```

Reply:
Done. The variables are now used instead of magic numbers.